spin-observatory-plugin-deck/src/components/PluginContainer.tsx

```diff
@@ -18,7 +19,8 @@ export function PluginContainer({ app }: IPluginContainerProps) {
   const [selectedPipeline, setSelectedPipeline] = useState<IPipeline>();
   const [selectedParams, setSelectedParams] = useState<string[]>([]);
   const [selectedDateRange, setSelectedDateRange] = useState<IDateRange>({ start: 0, end: MAX_DATE_RANGE });
-  const [selectedStatus, setSelectedStatus] = useState<string[]>(STATUSES);
+  const [selectedStatus, setSelectedStatus] = useState<string[]>([]);
+  const [statusCount, setStatusCount] = useState<any>({});
```

Comment:
Can this be something other than `any`? `any` should be a last resort.

Reply:
Yeah, it looks like I can use `Map<string, number>` here. I'll update it.